.gitlab-ci.yml

```diff
@@ -87,8 +87,8 @@ haddock:
     expire_in: 1 month
   script:
     - .ci/build_docs.sh
-  # Run on shared runners
-  tags:
+  # Do not run on shared runners
+  tags: [ local ]
```

Comment:
So the haddock job kept getting killed with code 137 (OOM), but the memory usage increase is somewhat hard to debug. So I hope we can all live with this.

Reply:
This exists on my CI changes branch anyway, so me and @martijnbastiaan can definitely live with it

Reply:
Jep